build/charts/antrea/templates/_helpers.tpl

```diff
@@ -18,8 +18,56 @@
 {{- end }}
```

Comment:
Will `antreaImageTag` continue to be used?

Reply:
yes as of now, we will be using it.